dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api.test/pages/project/WorkFlowDefinitionPage.java

```diff
@@ -0,0 +1,156 @@
+        params.put("pageNo",1);
+        params.put("searchVal",workFlowName);
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        RequestClient requestClient = new RequestClient();
+        ProjectPage project = new ProjectPage();
+        String projectCode = project.getProjectCode(sessionId, projectName);
+        HttpResponse res = requestClient.get("/projects/"+ projectCode + "/process-definition", headers, params);
+        System.out.printf("查询工作流结果：%s\n", res);
+        for (WorkFlowResponseTotalList workFlowList : JSONUtils.convertValue(res.body().data(), WorkFlowResponseData.class).totalList()) {
+            workFlowCode =  workFlowList.code();
+        }
+
+        return res;
+    }
+
+    public HttpResponse onLineWorkflow(String sessionId, String projectName, String workFlowName){
+        WorkFlowDefinitionPage workflow = new WorkFlowDefinitionPage();
+        workflow.queryWorkflow(sessionId, projectName, workFlowName);
+        Map<String, Object> params = new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
+        params.put("name", workFlowName);
+        params.put("releaseState", "ONLINE");
+        headers.put(Constants.SESSION_ID_KEY, sessionId);
+        RequestClient requestClient = new RequestClient();
+        ProjectPage project = new ProjectPage();
+        String projectCode = project.getProjectCode(sessionId, projectName);
+        HttpResponse res = requestClient.post("/projects/"+projectCode+"/process-definition/"+workFlowCode+"/release", headers, params);
+
+        System.out.printf("上线工作流：%s", res);
+        return res;
+
+    }
+
+    public HttpResponse runWorkflow(String sessionId, String projectName, String workFlowName){
+        WorkFlowDefinitionPage workflow = new WorkFlowDefinitionPage();
+        workflow.queryWorkflow(sessionId, projectName, workFlowName);
+        workflow.onLineWorkflow(sessionId, projectName, workFlowName);
+        Map<String, Object> params = new HashMap<>();
+        Map<String, String> headers = new HashMap<>();
+        params.put("processDefinitionCode", workFlowCode);
+        params.put("startEndTime", "2022-06-25T16:00:00.000Z");
+        params.put("startEndTime", "2022-06-25T16:00:00.000Z");
+        params.put("scheduleTime", "2022-06-26 00:00:00,2022-06-26 00:00:00");
+        params.put("failureStrategy", "CONTINUE");
+        params.put("warningType", "NONE");
+        params.put("warningGroupId", "");
+        params.put("execType", "START_PROCESS");
+        params.put("startNodeList", "");
+        params.put("taskDependType", "TASK_POST");
+        params.put("dependentMode", "OFF_MODE");
+        params.put("runMode", "RUN_MODE_SERIAL");
+        params.put("processInstancePriority", "MEDIUM");
+        params.put("workerGroup", "default");
+        params.put("environmentCode", "");
+        params.put("startParams", "");
+        params.put("expectedParallelismNumber", "");
+        params.put("dryRun", 0);
```

Comment:
It is recommended to create an entity class instead of a map to increase code readability and reusability.

Reply:
Thanks for your advice, I will fix it as soon as possible